app/errors.js

```diff
@@ -32,3 +32,6 @@ exports.authenticationError = message => internalError(message, exports.AUTHENTI
 
 exports.AUTHORIZATION_ERROR = 'authorization_error';
 exports.authorizationError = message => internalError(message, exports.AUTHORIZATION_ERROR);
+
+exports.FORBIDDEN_ERROR = 'forbidden_error';
```

Comment:
I think authorization and forbidden should be the same, what would be the difference?

Reply:
Yes is the same, i skipped fordbidden